src/OpenTK/Platform/X11/X11DisplayDevice.cs

```diff
@@ -297,6 +325,7 @@ private bool QueryXF86(List<DisplayDevice> devices)
                 dev.BitsPerPixel = FindCurrentDepth(currentScreen);
                 dev.RefreshRate = (pixelClock * 1000F) / (currentMode.vtotal * currentMode.htotal);
                 currentScreen++;
+                devices[i] = dev;
             }
```

Comment:
Leave a space after the final closing brace. It sets the return apart from the rest of the code.